charts/traceability-foss-backend/values.yaml

```diff
@@ -111,58 +115,52 @@ ingress:
   hosts: []
   tls: []
 
-##############################
-# Application Configuration  #
-#############################
-
-springprofile: <springprofile>
-
 healthCheck:
   enabled: true #<healthCheck.enabled>
 
 traceability:
-  bpn: <traceability.bpn>
-  url: <traceability.url>
+  bpn: "CHANGEME" #<traceability.bpn>
```

Comment:
CHANGEME should be removed as this is defined by property #<xy> This is covered in the admin manual so changeme does not provide any furter benefit for the admin. Use empty value of nothing.